lib/Extension/config/benchmark/executors.php

```diff
@@ -11,18 +11,6 @@
  */
 
 return [
-    'memory_centric_microtime' => [
-        'executor' => 'memory_centric_microtime',
-    ],
-    'remote' => [
-        'executor' => 'remote',
-    ],
-    'local' => [
-        'executor' => 'local',
-    ],
-    'debug' => [
-        'executor' => 'debug',
-    ],
```

Comment:
services can now be referenced directly by name without prior-configuration (and they will use their own defaults)